### PR TITLE
Update HE

### PR DESCRIPTION
**Pull request by RoyStegeman**
pt=100 GeV.
LO

HpT-MON
```
N=1.100000e+00: dHdpt = 2.371747e-04 +- 6.998981e-08. 
N=1.200000e+00: dHdpt = 2.222881e-04 +- 6.665811e-08. 
N=1.300000e+00: dHdpt = 2.096299e-04 +- 6.389373e-08. 
N=1.400000e+00: dHdpt = 1.987235e-04 +- 6.154140e-08. 
N=1.500000e+00: dHdpt = 1.892184e-04 +- 5.977419e-08. 
N=1.600000e+00: dHdpt = 1.808507e-04 +- 5.798244e-08. 
N=1.700000e+00: dHdpt = 1.734226e-04 +- 5.647294e-08. 
N=1.800000e+00: dHdpt = 1.667780e-04 +- 5.513436e-08. 
N=1.900000e+00: dHdpt = 1.607938e-04 +- 5.393029e-08. 
N=2.000000e+00: dHdpt = 1.553722e-04 +- 5.284757e-08. 
N=2.100000e+00: dHdpt = 1.504334e-04 +- 5.176628e-08. 
N=2.200000e+00: dHdpt = 1.459140e-04 +- 5.093982e-08. 
N=2.300000e+00: dHdpt = 1.417580e-04 +- 5.006537e-08. 
N=2.400000e+00: dHdpt = 1.379210e-04 +- 4.918402e-08. 
N=2.500000e+00: dHdpt = 1.343659e-04 +- 4.833186e-08. 
N=2.600000e+00: dHdpt = 1.310616e-04 +- 4.761258e-08. 
N=2.700000e+00: dHdpt = 1.279804e-04 +- 4.693863e-08. 
N=2.800000e+00: dHdpt = 1.250987e-04 +- 4.626544e-08. 
N=2.900000e+00: dHdpt = 1.223968e-04 +- 4.559626e-08. 
```

Hpt-N3LO
```
N=1.100000e+00: dHdpt = 2.502764e-04 + 0.000000e+00 II. 
N=1.200000e+00: dHdpt = 2.290222e-04 + 0.000000e+00 II. 
N=1.300000e+00: dHdpt = 2.127171e-04 + 0.000000e+00 II. 
N=1.400000e+00: dHdpt = 2.000515e-04 + 0.000000e+00 II. 
N=1.500000e+00: dHdpt = 1.897655e-04 + 0.000000e+00 II. 
N=1.600000e+00: dHdpt = 1.810654e-04 + 0.000000e+00 II. 
N=1.700000e+00: dHdpt = 1.734972e-04 + 0.000000e+00 II. 
N=1.800000e+00: dHdpt = 1.667958e-04 + 0.000000e+00 II. 
N=1.900000e+00: dHdpt = 1.607924e-04 + 0.000000e+00 II. 
N=2.000000e+00: dHdpt = 1.553656e-04 + 0.000000e+00 II. 
N=2.100000e+00: dHdpt = 1.504265e-04 + 0.000000e+00 II. 
N=2.200000e+00: dHdpt = 1.459081e-04 + 0.000000e+00 II. 
N=2.300000e+00: dHdpt = 1.417548e-04 + 0.000000e+00 II. 
N=2.400000e+00: dHdpt = 1.379190e-04 + 0.000000e+00 II. 
N=2.500000e+00: dHdpt = 1.343655e-04 + 0.000000e+00 II. 
N=2.600000e+00: dHdpt = 1.310627e-04 + 0.000000e+00 II. 
N=2.700000e+00: dHdpt = 1.279827e-04 + 0.000000e+00 II. 
N=2.800000e+00: dHdpt = 1.251028e-04 + 0.000000e+00 II. 
N=2.900000e+00: dHdpt = 1.224015e-04 + 0.000000e+00 II. 
N=3.000000e+00: dHdpt = 1.198627e-04 + 0.000000e+00 II. 

```

### CODE REVIEW

**Comment by Radonirinaunimi** [comment]
@RoyStegeman Nice! I will reflect these changes in Main instead since it produce results for LO+NLO. 

The only thing missing for the HE contribution is scale variation.

**Comment by RoyStegeman** [comment]
Well somehow I find much larger disagreement between FO and HE here than in mathematica, so I'm still trying to understand that.

**Comment by Radonirinaunimi** [comment]
> Well somehow I find much larger disagreement between FO and HE here than in mathematica, so I'm still trying to understand that.

Ok! I will wait for that.

File: src/CombinedRes.cpp [hunk 1]
```diff
@@ -69,16 +69,39 @@ std::complex<long double> CombinedRes::Matching(std::complex<long double> N,
 
 std::complex<long double> CombinedRes::CombinedResExpr(
     std::complex<long double> N, long double pt, int scheme) {
+  double pp = static_cast<double>(pt);
+  // take only real part. Does not work for complex
+  double nn = static_cast<double>(N.real());
   std::complex<long double> mres;
+  std::vector<double> ResultsMellin;
+  std::vector<double> zero(2, 0.0);
+
```
Comment (Radonirinaunimi) [inline src/CombinedRes.cpp:78]: What does setting SigmaLO from hpt-mon to 2 mean here?

Comment (RoyStegeman) [reply src/CombinedRes.cpp:78]: 2 is the length of the vector

Comment (Radonirinaunimi) [reply src/CombinedRes.cpp:78]: OMG, I was being stupid here. I wrote it myself and forgot what it meant :rofl: I thought it was a complex number.

**Comment by RoyStegeman** [comment]
Shall we merge both PRs? These PRs feel a bit chaotic and I think we're a good checkpoint.

**Pull request merged**